tests/unit/Application/Metadata/JsonGeneratorTest.php

```diff
@@ -92,7 +95,7 @@ public function test_it_can_build_saml_entity_metadata_for_new_entities()
         $this->assertEquals('http://metadata', $metadata['metadataurl']);
         $this->assertEquals('testaccepted', $metadata['state']);
         $this->assertEquals('saml20-sp', $metadata['type']);
-        $this->assertEquals('revisionnote', $metadata['revisionnote']);
+        $this->assertMatchesRegularExpression('/Entity Created by user John Doe with email address "jd@example.com"\nVia the SPdashboard on .* \nComment: "revisionnote"/', $metadata['revisionnote']);
```

Comment:
Nice one, never seen used before

Reply:
I can be very usefull when certain parts of a match are variable (like the timestamp) :+1: one for your toolbelt